src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,57 @@
+package qna.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question", cascade = CascadeType.PERSIST)
+    private final List<Answer> answers;
+
+    protected Answers() {
+        this.answers = new ArrayList<>();
+    }
+
+    public Answers(Answer answer) {
+        this.answers = new ArrayList<>();
+        this.answers.add(answer);
```

Comment:
```suggestion
        addAnswer(answer);
```
내부 메서드를 호출하면 일관되게 answer를 추가할수 있겠네요!